### ISSUE TITLE
Where is the topic vector based on VAE

### ISSUE THREAD

**Issue by LeMei** [something678/TodKat#16]
I can not find the topic vector code based on VAE. Would someone find it?

**Comment by MrZilinXiao** [comment]
I believe it is located here: https://github.com/something678/TodKat/blob/1f711496459c521796043f07861a259c6fe37df5/src/roberta_with_finetune.py#L191

However, no trace of topic pretaining VAE code was found.

**Comment by MrZilinXiao** [comment]
@LeMei  Sorry that I made a mistake. `get_topic_vec` is not the topic encoder mentioned in the paper. I believe the code is missing crucial parts of the paper's contributions.

**Comment by caolanying** [comment]
> @LeMei Sorry that I made a mistake. `get_topic_vec` is not the topic encoder mentioned in the paper. I believe the code is missing crucial parts of the paper's contributions.
yes,you are right,there is no topic process...(after spending 2 hours finding it)
